react/features/base/premeeting/components/web/Preview.tsx

```diff
@@ -86,16 +87,16 @@ function Preview(props: Props) {
  * @param {Props} ownProps - The own props of the component.
  * @returns {Props}
  */
-function _mapStateToProps(state, ownProps) {
+function _mapStateToProps(state: IReduxState, ownProps: Partial<Props>) {
```

Comment:
could we go with useSelector approach instead?

Reply:
We could, but that's not a TS change; it's a refactor for the component